notte/env.py

```diff
@@ -119,39 +141,63 @@ async def execute(
         logger.info(f"🌌 action {action_id} executed in browser")
         return self._preobserve(snapshot)
 
+    @timeit("execute_special")
+    async def _execute_special(
```

Comment:
In the same way that we have an `ExecutionPipe` that allows to hide the boilerplate in this `env.py` file— i would do the same for the special actions